### ISSUE TITLE
Update doc for `.krmignore` behavior

### ISSUE THREAD

**Issue by Shell32-Natsu** [GoogleContainerTools/kpt#1733]
We should update the doc website to document the `.krmignore` file behavior in kpt commands.

**Comment by Shell32-Natsu** [comment]
cc @mikebz @frankfarzan

**Comment by mortent** [comment]
There are still some questions around whether the `.krmignore`file is the correct way to handle this. Ref https://github.com/GoogleContainerTools/kpt/issues/1664

**Comment by Shell32-Natsu** [comment]
@mortent Yes but we should document it for *current* behavior. If you think it's unnecessary then we can close this.

**Comment by aravindtga** [comment]
Hi @vickywane, are you working on this? Happy to pick it up if not?
